extensions/visual-studio-code-snippet-manager/src/data.ts

```diff
@@ -0,0 +1,23 @@
+export const path = `${process.env.HOME}/Library/Application Support/Code/user/snippets/raycast.code-snippets`;
+export const languages = [
```

Comment:
mind adding [all of the language identifiers](https://code.visualstudio.com/docs/languages/identifiers#_known-language-identifiers)? 😄

Reply:
Thank you!
I was looking for that info.

I'll fix it and give you a PR.